core/config.go

```diff
@@ -88,3 +93,103 @@ type Config struct {
 	// authentication method
 	PatchBeanstalkUserdata string
 }
+
+// ParseConfig loads configuration from command line flags, environments variables, and config files.
+func ParseConfig(conf *Config) {
```

Comment:
Renamed from `parseCommandLineFlags` since it loads config from environment variables and config files too.